setup.py

```diff
@@ -158,10 +161,12 @@ def build_extension(self, ext):
     name="pyvelox",
     version=VERSION,
     description="Python bindings and extensions for Velox",
+    long_description=readme,
+    long_description_content_type="text/markdown",
     url="https://github.com/facebookincubator/velox",
     author="Meta",
     author_email="velox@fb.com",
-    license="BSD",
+    license="Apache License 2.0",
```

Comment:
I think TA was released with BSD license - what is motivation for this change ?

Reply:
Velox overall is licensed as Apache 2.0 :)